lib/checknullpointer.h

```diff
@@ -138,6 +138,7 @@ class CPPCHECKLIB CheckNullPointer : public Check {
      * Dereferencing a pointer and then checking if it's NULL..
      */
     void nullPointerByDeRefAndChec();
```

Comment:
Not your code, but this should probably be `nullPointerByDerefAndCheck()`. Also applies to the new function.